workflow/envs/nextstrain.yaml

```diff
@@ -2,6 +2,7 @@ name: nextstrain
 channels:
   - bioconda
   - defaults
+  - conda-forge
```

Comment:
To make sure [dependencies that exist in multiple channels get sourced correctly](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-channels.html), `conda-forge` needs to be the first channel listed like:

```yaml
channels:
  - conda-forge
  - bioconda
  - defaults
```

The order dependency is annoying, but it will save us some headaches in the future to use this order.

Reply:
Ahhh thank you for the conda pro tip!